src/api.ts

```diff
@@ -1,11 +1,25 @@
-import { FluenceClient } from './FluenceClient';
-import { SecurityTetraplet } from './internal/commonTypes';
-import { Particle } from './internal/particle';
 import Multiaddr from 'multiaddr';
-import PeerId, { isPeerId } from 'peer-id';
-import { generatePeerId, seedToPeerId } from './internal/peerIdUtils';
-import { FluenceClientImpl } from './internal/FluenceClientImpl';
-import log from 'loglevel';
+import PeerId from 'peer-id';
+import { PeerIdB58, SecurityTetraplet } from './internal/commonTypes';
+import * as unstable from './api.unstable';
```

Comment:
maybe rename `unstable` as `new`? or some other positive term

Reply:
I thin `unstable` conveys the message better